action.yaml

```diff
@@ -1,11 +1,25 @@
-
 name: 'Qodana - Code Inspection'
-description: 'Qodana is a code quality monitoring tool to identify and suggest fixes for bugs, security vulnerabilities, duplications, and imperfections.
-It brings all the smart features you love in the IntelliJ IDEs into your project pipelines.'
+description: 'Qodana is a code quality monitoring tool that identifies and suggests fixes for bugs, security vulnerabilities, duplications, and imperfections.'
 author: 'JetBrains'
 
+inputs:
+  project-dir:
+    description: 'Project folder to inspect'
+    required: false
+    default: '/github/workspace'
+  results-dir:
+    description: 'Save results to folder'
+    required: false
+    default: 'qodana'
+  report-dir:
+    description: 'Save html report to folder'
+    required: false
+    default: 'report'
 runs:
-  using: "composite"
-  steps:
-    - run: docker run -v ${GITHUB_WORKSPACE}:/data/project -v ${GITHUB_WORKSPACE}/qodana:/data/results jetbrains/qodana:2020.3-eap --save-report
-      shell: sh
+  using: 'docker'
+  image: 'docker:://jetbrains/qodana:latest'
```

Comment:
I believe it should be one column
`docker://`

Reply:
well spotted!